### ISSUE TITLE
can't be builded with flutter 3 - flutter_native_timezone want old Gradle plugin

### ISSUE THREAD

**Issue by Akiyamka** [cph-cachet/carp.sensing-flutter#326]
```
> flutter build apk 
* What went wrong:
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.5.20 and higher.
The following dependencies do not satisfy the required version:
project ':flutter_native_timezone' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.50

Flutter 3.10.5 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 796c8ef792 (2 weeks ago) • 2023-06-13 15:51:02 -0700
Engine • revision 45f6e00911
Tools • Dart 3.0.5 • DevTools 2.23.1

**Comment by Akiyamka** [comment]
Looks like flutter_native_timezoneis not updated anymore, and better to use fork
https://github.com/pinkfish/flutter_native_timezone/issues/56

**Comment by bardram** [comment]
Hi @Akiyamka - I have noticed that `flutter_native_timezone` is no longer supported and I've switched to using [flutter_timezone](https://pub.dev/packages/flutter_timezone) instead. 

I am in the process of publishing a new major version of CAMS (version 1.1.0) where this (and a lot of other things) will be updated.

**Comment by bardram** [comment]
This has now been released as part of CAMS version 1.1.x

**Comment by Akiyamka** [comment]
Cool, thank you!
